controllers/users.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const { encode } = require('../utils');
+let db = require('../services/storage'),
+  encrypt = require('../services/encrypt'),
```

Comment:
Looks like these two variables can be const:
`const db = require('../services/storage'),`
` encrypt = require('../services/encrypt'),`